### PR TITLE
skip integrations that we don't have instance to them

### PR DESCRIPTION
**Pull request by anara123**
## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
2nd pull request to https://github.com/demisto/content/pull/6539
related to https://github.com/demisto/etc/issues/23712

## Description
skips integrations that for some reason don't have test playbooks, have no instance and don't appear in the conf.json

### CODE REVIEW

File: Tests/conf.json [hunk 1]
```diff
@@ -2670,10 +2673,27 @@
...
     },
     "skipped_integrations": {
         "_comment": "~~~ NO INSTANCE ~~~",
+        "URLhaus": "No instance",
+        "AlphaSOC Network Behavior Analytics": "No instance",
+        "IsItPhishing": "No instance",
+        "Verodin": "No instance",
+        "EasyVista": "No instance",
+        "Pipl": "No instance",
+        "Moloch": "No instance",
```
Comment (ronykoz) [inline Tests/conf.json:2689]: are you sure about that? they have a public demo we can test on

**Comment by ShahafBenYakir** [comment]
Do we have open issues for those? Are we waiting for an instance or we are not even planning on getting one?

**Comment by anara123** [comment]
@ShahafBenYakir @ronykoz I didn't their instances in our conf-test repo. And some of them probably have public instance, API Key. But till now those integrations weren't mentioned in the conf.json. I don't think we need to file issues for those. When we will have bug/enhancement we may do it.
And this PR is about to fix the conf.json/content validation. And not unskip test playbooks

**Comment by anara123** [comment]
@ShahafBenYakir can we merge this PR?

**Pull request merged**